src/target/ext/base/multithread.rs

```diff
@@ -186,6 +200,39 @@ pub trait MultiThreadOps: Target {
     }
 }
 
+/// Target Extension - [Reverse continue] for multi threaded targets.
+///
+/// Reverse continue allows the target to run backwards until it reaches the end
+/// of the replay log.
+///
+/// [Reverse continue]: https://sourceware.org/gdb/current/onlinedocs/gdb/Reverse-Execution.html
+pub trait MultiThreadReverseCont: Target + MultiThreadOps {
+    /// Reverse-continue the target.
+    fn reverse_cont(
+        &mut self,
+        check_gdb_interrupt: &mut dyn FnMut() -> bool,
+    ) -> Result<ThreadStopReason<<Self::Arch as Arch>::Usize>, Self::Error>;
+}
+
+define_ext!(MultiThreadReverseContOps, MultiThreadReverseCont);
+
+/// Target Extension - [Reverse stepping] for multi threaded targets.
+///
+/// Reverse stepping allows the target to run backwards by one step.
+/// Usually this corresponds to a single instruction.
```

Comment:
...just remove this line entirely.
The term `step` is entirely sufficient.

Reply:
Oops. I could've sworn I removed at least one of these... how many times did I repeat this? 😅